frenchchocolatine.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Frenchchocolatine",
+  "logoUrl": "https://github.com/frenchchoco/validator-registry/blob/main/genesis-frenchchoco.jpg",
```

Comment:
not the good URL for the raw file, it displays the GitHub UI, just use the same path than the URL in my merged PR.